lib/src/components/NxTab/NxTab.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
```

Comment:
This whole file is inconsistent in its use of curly-braces spacing – some have spaces, some others don't. I don't feel strongly about either style as long as the file itself uses just one.

Reply:
> This whole file is inconsistent in its use of curly-braces spacing – some have spaces, some others don't. I don't feel strongly about either style as long as the file itself uses just one.

Actually not only this file, but many others:
* lib/src/components/NxTabs/NxTabs.tsx
* lib/src/components/NxTab/__tests__/NxTab.test.tsx
* lib/src/components/NxTabPanel/NxTabPanel.tsx
* lib/src/components/NxTabPanel/__tests__/NxTabPanel.test.tsx

Reply:
I feel like we should have an eslint rule for this.

Reply:
> I feel like we should have an eslint rule for this.

Does it exist? If it does I'm +1 on having it.

Reply:
f5af121

Reply:
> Does it exist? If it does I'm +1 on having it.

We've discussed this multiple times, people on the team have differing preferences on it and none of them feel strongly enough to try to push out the others.  While it is a little odd that there were inconsistencies in brand new files written by a single person, I wouldn't have said it was important to rectify.

Reply:
I think I lean toward no space, but VSC added the space sometimes during copy/paste.

Reply:
ran the VSC formatter on the files in lib and fixed any remaining issues reported by eslint 8b086d6